typings/organization.d.ts

```diff
@@ -10,14 +10,16 @@ export interface OrganizationProp {
 }
 
 export interface ContentfulOrganizationAPI {
-  createAppDefinition(data: AppDefinitionProps): Promise<AppDefinition>,
-  getAppDefinition(id: string): Promise<AppDefinition>,
-  getAppDefinitions(): Promise<Collection<AppDefinition>>,
-  createTeamMembership(data: TeamMembershipProps): Promise<TeamMembership>,
-  getTeamMembership(teamId: string, id: string): Promise<TeamMembership>,
-  getTeamMembership(teamId: string): Promise<Collection<TeamMembership>>,
-  getOrganizationInvitation(id: string): Promise<OrganizationInvitation>,
-  createOrganizationInvitation(data: OrganizationInvitationProps): Promise<OrganizationInvitation>
+  createAppDefinition(data: AppDefinitionProps): Promise<AppDefinition>;
+  getAppDefinition(id: string): Promise<AppDefinition>;
+  getAppDefinitions(): Promise<Collection<AppDefinition>>;
+  createTeamMembership(data: TeamMembershipProps): Promise<TeamMembership>;
+  getTeamMembership(teamId: string, id: string): Promise<TeamMembership>;
+  getTeamMemberships(opts: object): Promise<Collection<TeamMembership>>;
```

Comment:
this opts object should be optional and could reference a more specific type (opts might contain `teamID` and/or `query`, right? but both optional?)